src/pages/docs/administration/retention-policies/retention-policy-tentacle-cleanup-and-troubleshooting.md

```diff
@@ -115,26 +116,22 @@ See below the messages you will have in your raw deployment logs at the end of a
 You can find your packages in the `<Tentacle Home>\Files` directory.
 or `<Tentacle Home>\[Instance Name])\Files` if you have multiple 
 Tentacle instances on one machine.
-- For Windows Tentacles, the default directory is: `C:\Octopus\Files`
```

Comment:
Why was this removed?

Reply:
It's already mentioned in lines 21-26 (doesn't show up in the PR since I didn't change those lines).